src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: "#{$i}";
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
```

Comment:
- Don't use SASS loops for styles that stay the same for all elements of the group

Reply:
will do